docs/performance/getting-started.md

```diff
@@ -0,0 +1,73 @@
+# Getting started with Performance Monitoring
+
+##  Basic usage
+
+**TBD** basic explainer
+
+```ts
+constructor(private afp: AngularFirePerformance, private afs: AngularFirestore) {}
+
+ngOnInit() {
+  this.articles = afs.collection('articles')
+      .collection('articles', ref => ref.orderBy('publishedAt', 'desc'))
+      .snapshotChanges()
+      .pipe(
+        this.afp.trace('getArticles'),
+        map(articles => ...)
+      );
+}
+```
+
+`trace(name:string)` will trace the time it takes for your observable to either complete or emit it's first value. Beyond the basic trace there are three other operators:
+
+### `traceComplete(name:string)`
```

Comment:
to me traceComplete sounds like I'm telling the trace to finish. maybe traceCompletion or traceUntilComplete? if traceUntil, then we could have trace, traceUntil, traceUntilComplete, traceUntilFirst

Reply:
How about `traceLast` if we go with `traceFirst`, as that matches the `takeLast` operator. I was roughly aiming to have them follow existing rxjs naming conventions. 

Maybe add a `traceWhile` which is the opposite of `traceUntil` too.

Thoughts @davideast?

Reply:
Though it is really an until now that I think about it, because it's hot... how about `trace`, `traceUntil`, `traceUntilLast`, `traceUntilFirst`, `traceWhile`?

Reply:
Went with that naming convention.